src/features/cards/organisms/card-item.js

```diff
@@ -76,3 +86,48 @@ const CardBox = styled(Card)`
   max-height: 24rem;
   overflow-y: hidden;
 `
+
+const PopUpButton = () => {
```

Comment:
Please, rename it to `DeleteButton`

Reply:
Renamed it to CardDeletePopUpButton as you suggested in #46 Which one we keep?